CHANGELOG.md

```diff
@@ -1298,6 +1298,15 @@ Deprecated names
 * This fixes the fact we had picked the wrong name originally. The erased modality
   corresponds to @0 whereas the irrelevance one corresponds to `.`.
 
+### Deprecated `Relation.Binary.PropositionalEquality.inspect`
+    in favour of `with ... in ...` syntax (issue #1580; PRs #1630, #1930)
+
+* In `Relation.Binary.PropositionalEquality`
+  both the record type `Reveal_·_is_`
+  and its principal mode of use, `inspect`,
+  have been deprecated in favour of the new `with ... in ...` syntax.
```

Comment:
It would be good to have a link to the Agda docs here, and maybe a very simple example?

Reply:
There's a `README.WithIn` now?

Reply:
Added the link... but I think a 'simple' example isn't easily expressible in the `CHANGELOG`. Suggestions?